{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -48,6 +48,20 @@
     },
 }
 
+
+REDIS_URL = env("REDIS_TLS_URL", default=env("REDIS_URL"))
+REDIS_SSL = env.bool("REDIS_SSL", default=False)
+
+CERT_NONE = 0
+
+if REDIS_SSL:
+    CELERY_REDIS_BACKEND_USE_SSL = {"ssl_cert_reqs": CERT_NONE}
+    CELERY_BROKER_USE_SSL = {"ssl_cert_reqs": CERT_NONE}
```

Comment:
Can you move these next to the other celery settings in `base.py`? Try to keep backend and broker related settings together please and link to the celery docs for each. That should give something like this:


```diff
  # Celery
  # ------------------------------------------------------------------------------
  if USE_TZ:
      # https://docs.celeryq.dev/en/stable/userguide/configuration.html#std:setting-timezone
      CELERY_TIMEZONE = TIME_ZONE
  # https://docs.celeryq.dev/en/stable/userguide/configuration.html#std:setting-broker_url
  CELERY_BROKER_URL = REDIS_URL
+ # https://docs.celeryq.dev/en/stable/userguide/configuration.html#redis-backend-use-ssl
+ CELERY_BROKER_USE_SSL = {"ssl_cert_reqs": ssl.CERT_NONE} if REDIS_SSL else None
  # https://docs.celeryq.dev/en/stable/userguide/configuration.html#std:setting-result_backend
  CELERY_RESULT_BACKEND = REDIS_URL
+ # https://docs.celeryq.dev/en/stable/userguide/configuration.html#redis-backend-use-ssl
+ CELERY_REDIS_BACKEND_USE_SSL = CELERY_BROKER_USE_SSL
  # https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-extended
  CELERY_RESULT_EXTENDED = True
```